packages/vue/src/components/molecules/SfSelect/SfSelect.vue

```diff
@@ -227,6 +223,22 @@ export default {
     update(index) {
       this.index = index;
     },
+    addOptionsAndIndexes() {
+      const options = [];
+      const indexes = {};
+      if (!this.$slots.default) return;
+      this.$on("update", this.update);
+      this.$slots.default.forEach((slot, index) => {
```

Comment:
Maybe slot could be destructured, wdyt? :)

Reply:
done :)